README.md

```diff
@@ -35,11 +35,29 @@ See [contributing instructions](CONTRIBUTING.md) to help improve this project.
   * [Group migration workflow](#group-migration-workflow)
   * [Debug notebook](#debug-notebook)
   * [Debug logs](#debug-logs)
-  * [Table Migration Workflow](#table-migration-workflow)
-    * [Dependency CLI commands](#dependency-cli-commands)
-    * [Table Migration Workflow Tasks](#table-migration-workflow-tasks)
+  * [Table Migration](#table-migration)
+    * [Prerequisites](#prerequisites)
+    * [Upgrade Process](#upgrade-process)
+      * [Step 1: Mapping Metastore Tables (UCX Only)](#step-1-mapping-metastore-tables-ucx-only)
+        * [Step 1.1: Create the mapping file](#step-11-create-the-mapping-file)
+        * [Step 1.2: Update the mapping file](#step-12-update-the-mapping-file)
+      * [Step 2: Create the necessary cloud principals for the upgrade](#step-2-create-the-necessary-cloud-principals-for-the-upgrade)
+        * [Step 2.1: Map the cloud principals to the cloud "prefixes"](#step-21-map-the-cloud-principals-to-the-cloud-prefixes)
+        * [Step 2.2: Create/Modify Cloud Principals and Credentials](#step-22-createmodify-cloud-principals-and-credentials)
+        * [Step 2.3: Create External Locations](#step-23-create-external-locations)
+        * [Step 2.4: Create "Uber Principal"](#step-24-create-uber-principal)
+        * [Step 2.5: Create Catalogs and Schemas](#step-25-create-catalogs-and-schemas)
+    * [Step 3: Upgrade the Metastore](#step-3-upgrade-the-metastore)
+    * [Step 4: Odds and Ends](#step-4-odds-and-ends)
+      * [Step 4.1: Skipping Table/Schema](#step-41-skipping-tableschema)
+      * [Step 4.2: Moving objects](#step-42-moving-objects)
+      * [Step 4.2: Aliasing objects](#step-42-aliasing-objects)
+      * [Step 4.3: Reverting objects](#step-43-reverting-objects)
```

Comment:
I'm not sure these belong in the ToC which is supposed to be an overview. 

Although there are #anchors available for every heading, that doesn't mean the ToC needs to and arguably these headings are content (and not headings for the content).

Reply:
The TOC is autogenerated includes all subheadings. The autogeneration has my preference over manually excluding the `.x` steps, though, if could configure it we can exclude this level